### DISCUSSION TITLE
"All systems are operational" twice on GitHub Pages with "tcp-ping" as check

### DISCUSSION THREAD

**Discussion by thomasmerz** [upptime/upptime#272] [General] [answered]
Since I use the new check "tcp-ping" I have two lines "All systems are operational" on my GH Page.  
❓ Is this intended or would it be possible to have only one line because if all systems/checks are "ok" than nobody needs this info twice…

**Comment by thomasmerz** [comment]
Relates https://github.com/upptime/upptime/issues/270

**Comment by ririko5834** [comment]
Same problem for me and I even don't use TCP ping

**Comment by AnandChowdhary** [comment, marked as answer]
This is now fixed, thanks for the quick report!
